models/labels/addresses/institution/identifier/cex/labels_cex_bitcoin.sql

```diff
@@ -7,7 +7,7 @@
                                     \'["hildobby"]\') }}')}}
 
 SELECT blockchain
-, address
+, from_base58(address) as address
```

Comment:
Seems there are invalid address in the ref table:
https://github.com/duneanalytics/spellbook/actions/runs/5595866970/jobs/10232198153?pr=3789#step:12:27

Reply:
The query shows `from_base58()` receiving invalid inputs:
https://dune.com/queries/2753286